src/lightning/fabric/utilities/throughput.py

```diff
@@ -380,6 +394,11 @@ def get_available_flops(device: torch.device, dtype: torch.dtype) -> Optional[in
             # if we were able to parse the chip, it should be in the flops list
             raise RuntimeError(f"FLOPs not found for {device_name!r}, chip is {chip!r}")
         dtype_to_flops = _CUDA_FLOPS[chip]
+        if dtype is torch.float32:
+            from lightning.fabric.accelerators.cuda import _is_ampere_or_later
```

Comment:
Can you move it to a global import and combine the if-conditions

Reply:
I get circular import errors. Accelerators import utilities, so utilities cannot import accelerators